ci-operator/config/stolostron/cluster-curator-controller/stolostron-cluster-curator-controller-backplane-2.3.yaml

```diff
@@ -11,7 +11,7 @@ build_root:
 images:
 - dockerfile_path: Dockerfile.prow
   inputs:
-    stolostron_builder_go1.20-linux:
+    stolostron_builder_go1.18-linux:
       as:
```

Comment:
Will auto update upgrade it to 1.19 automatically?

Reply:
Sounds good!

Reply:
I'd rather actually update it here, just in case it doesn't, so we don't have to open another PR. The auto upgrade should be able to ignore the change if otherwise.

Reply:
Better to be safe...